README.md

```diff
@@ -1,10 +1,31 @@
 # Pulplift
 
-This is based on and makes use of the Vagrant based configuration of the [Forklift](https://github.com/theforeman/forklift) project. Please see the Forklift documentation for all configuration options available when creating custom or adding new boxes.
+This is based on and makes use of the Vagrant based configuration of the
+[Forklift](https://github.com/theforeman/forklift) project. Please see the Forklift documentation
+for all configuration options available when creating custom boxes.
+
+## Quickstart
+
+To setup Pulplift, ensure that all submodules are updated and in place.
+
+```
+git submodule update --init
+```
+
+You will also need to [clone the source
+code](https://docs.pulpproject.org/en/3.0/nightly/contributing/dev-setup.html#get-the-source).
+
+Finally, you can spin up your development environment. With ``vagrant up <box-name>``
```

Comment:
add a note here to remind that first config file should be copied before running vagrant up

Reply:
It doesn't need to be actually. If you haven't copied the config file, it will use the appropriate example file (which will install pulp_file)